man/man8/zfs.8

```diff
@@ -5354,7 +685,8 @@ See the
 of the
 .Xr smb.conf 5
 man page for all configuration options in case you need to modify any options
-to the share afterwards. Do note that any changes done with the
+to the share afterwards.
+Do note that any changes done with the
 .Xr net 8
 command will be undone if the share is ever unshared (such as at a reboot etc).
 .El
```

Comment:
In the `SEE ALSO` section that follows, should we add all the new subcommand manpages?

Reply:
I think that would be clutter, as all the manpage names in the `DESCRIPTION` section will be hyperlinked (in HTML renderings, etc.). Is there any other special handling of the `SEE ALSO` section that it would benefit listing all the subcommand pages?

Reply:
That said, I *do* think that the `SEE ALSO` sections of the subcommand pages need some attention, and I'll work on a commit for that before this PR is merged.

Reply:
OK.  I'm not aware of any programmatic handling of the "see also" section.